### PR TITLE
Feature/version updates

### PR DESCRIPTION
**Pull request by fewknow**
Updated according to changes in MSSQL container, TF Providers.   Got demo working again.

### CODE REVIEW

**Comment by fewknow** [comment]
@copeia  Can you review this for me?

File: README.md [hunk 1]
```diff
@@ -87,6 +89,6 @@ To generate a token use
 
 Note: If you're using a Linux machine the first step is to give `rwxrwxrwx` permissions recursively to the `data` and `config` folders as Docker is unable to access them on a Linux machine without those permissions. The problem is not present on MacOS. Run `chmod -R 777 _data config`.
 
-Note: On MacOS you need to install `gnu-sed` to be able to run the `token_replacer` script, this is required because the version of `sed` bundbled with MacOS is different to the one that comes with Linux systems, this is documented [here](https://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed).
+Note: On MacOS you need to install `gnu-sed` and `jq` to be able to run the `token_replacer` script, this is required because the version of `sed` bundbled with MacOS is different to the one that comes with Linux systems, this is documented [here](https://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed).
```
Comment (copeia) [inline README.md:92]: Dig the updates here

File: terraform/apps/README.md [hunk 2]
```diff
@@ -13,17 +13,16 @@ Schemas are translated to a Terraform map through the policy-template file
...
                     {
                         "name": "application-portfolio-management",
                         "certificate-common-names":[
-                            "application-portfolio-management.dev.cloud.qvcdev.qvc.net",
-                            "application-portfolio-management.qa.cloud.qvcdev.qvc.net"
+                            "fewknow.net",
                         ],
                         "database":[
-                            "application-portfolio-management-role"
+                            "databse"
```
Comment (copeia) [inline terraform/apps/README.md:25]: Typo?

**Review by copeia — APPROVED** [review]
Looks great dude - seems like you've cleaned it up a lot.

File: scripts/quick_start.sh [hunk 3]
```diff
@@ -457,6 +457,7 @@ function reset_local(){
         # Clearing Mimir Project Certs
         printf "\e[0;34m\n\nClearing local cluster and project created certs\e[0m\n"
         for file in $(find ${PROJECT_ROOT}/config -type f -not -name "*.hcl" | sed s@//@/@); do
+            printf "Removeing file : ${file}\n"
```
Comment (copeia) [inline scripts/quick_start.sh:460]: Typo here

**Pull request merged**